src/bin/postinstall.js

```diff
@@ -3,12 +3,15 @@ const fs = require('fs');
 const crypto = require('crypto');
 const path = require('path');
 const { v4: uuidv4 } = require('uuid');
-const {PYTHAGORA_METADATA_DIR, CONFIG_FILENAME, PYTHAGORA_API_SERVER} = require("@pythagora.io/js-code-processing").common;
+const { PYTHAGORA_METADATA_DIR, CONFIG_FILENAME, PYTHAGORA_API_SERVER } = require("@pythagora.io/js-code-processing").common;
 const packageJson = require('../../package.json');
 
 const pythagoraVersion = packageJson.version;
 const configPath = path.join(process.cwd(), '../..', PYTHAGORA_METADATA_DIR, CONFIG_FILENAME);
 
+// Check if telemetry is enabled using an environment variable
+const telemetryEnabled = process.env.PYTHAGORA_TELEMETRY_ENABLED !== 'false';
```

Comment:
To avoid the double negative, please consider:

```
const telemetryEnabled = process.env.PYTHAGORA_TELEMETRY_DISABLED == true;
```

This allows for:

- `PYTHAGORA_TELEMETRY_DISABLED=1`
- `PYTHAGORA_TELEMETRY_DISABLED=true`
- `PYTHAGORA_TELEMETRY_DISABLED=True`

Reply:
Done